ansible/roles/test/tasks/wr_arp.yml

```diff
@@ -53,15 +57,15 @@
     ready_timeout: 240
 
 - name: Find appropriate VXLAN sender
-  shell: ip route show proto zebra type unicast | sed -e '/default/d' -ne '/0\//p' | head -n 1 | sed -ne 's/0\/.*$/1/p'
+  shell: ip route show proto 186 type unicast | sed -e '/default/d' -ne '/0\//p' | head -n 1 | sed -ne 's/0\/.*$/1/p'
```

Comment:
does the change works with 2018Nov image?

Reply:
I have tested this only with the master branch. 
For 2018Nov there is a separate branch, so if needed these changes could be cherry-picked.

Reply:
I think we should make change which will work for both zebra (which uses `proto zebra` and frr which uses `proto 186`)

Reply:
Done